user_query_optimizer/optimizations/parquet_ordering.py

```diff
@@ -0,0 +1,29 @@
+from clickhouse_cli.ui.parseutils.tables import extract_tables
+from sqlparse.sql import Where, Comparison
+
+
+# Optimization # 4
```

Comment:
Looks like we need to merge the parquet ordering change and rebase before merging this.

Reply:
ok i've removed the dist/* files! I've also made changes to the parquet ordering, so when you get a chance to review it, I can merge that